### PR TITLE
📝 add toggle documentation

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

**Review by jerembdn — APPROVED** [review]

File: docs/pages/docs/components/toggle.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import { Toggle } from "@tonightpass/kitchen"
+```
+
+## Default
+
+<Playground
+  code={`
+  () => {
+  const [checked, toggle] = useCheckbox();
+  const [checked1, toggle1] = useCheckbox(true);
```
Comment (AntoineKM) [inline docs/pages/docs/components/toggle.mdx:26]: this is second checked so replace `[checked1, toggle1]` by `[checked2, toggle2]`

File: docs/pages/docs/components/toggle.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+return (
+<Container gap={10}>
```
Comment (AntoineKM) [inline docs/pages/docs/components/toggle.mdx:28]: the indent is incorrect please fix it with tabulations

File: docs/pages/docs/components/toggle.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+`}
+/>
+
+## Sizes
+
+<Playground
+  code={`
+  () => {
+  const [checked, toggle] = useCheckbox();
+  const [checked1, toggle1] = useCheckbox();
```
Comment (AntoineKM) [inline docs/pages/docs/components/toggle.mdx:63]: same here

File: docs/pages/docs/components/toggle.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+return (
```
Comment (AntoineKM) [inline docs/pages/docs/components/toggle.mdx:64]: ident error

**Review by AntoineKM — CHANGES_REQUESTED** [review]

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**